credentials/utopia-natcert-vcdm-v2/credential.json

```
@@ -0,0 +1,81 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://w3id.org/citizenship/v4rc1",
+    "https://w3id.org/security/data-integrity/v2",
+    "https://contexts.vcplayground.org/examples/utopia-natcert-vcdm-v2/v1.json"
```

Use the render method context (which was setup yesterday).  The one here is also missing `mediaType`.
```suggestion
    "https://w3id.org/vc/render-method/v2rc1"
```